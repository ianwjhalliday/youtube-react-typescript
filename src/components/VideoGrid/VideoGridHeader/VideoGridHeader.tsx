import React from 'react';
import './VideoGridHeader.scss';

export interface VideoGridHeaderProps {
  title: string;
}

export function VideoGridHeader(props: VideoGridHeaderProps) {
  return (
    <div className="video-grid-header">
      <span className="title">{props.title}</span>
    </div>
  )
}

export default VideoGridHeader;
