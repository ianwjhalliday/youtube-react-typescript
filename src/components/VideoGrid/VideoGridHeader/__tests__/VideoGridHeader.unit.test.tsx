import React from 'react';
import {shallow} from 'enzyme';
import VideoGridHeader from '../VideoGridHeader';

test('renders VideoGridHeader', () => {
  const wrapper = shallow(
    <VideoGridHeader title="title" />
  );
  expect(wrapper).toMatchSnapshot();
});
