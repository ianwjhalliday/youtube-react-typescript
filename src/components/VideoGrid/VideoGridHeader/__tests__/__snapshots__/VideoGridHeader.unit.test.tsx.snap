// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders VideoGridHeader 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    title
  </span>
</div>
`;
